contracts/utils/ICS02ClientUpgradeable.sol

```diff
@@ -153,7 +171,7 @@ abstract contract ICS02ClientUpgradeable is IICS02Client, IICS02ClientErrors, Ac
     /// @inheritdoc IICS02Client
     function submitMisbehaviour(string calldata clientId, bytes calldata misbehaviourMsg) external {
         getClient(clientId).misbehaviour(misbehaviourMsg);
-        emit ICS02MisbehaviourSubmitted(clientId, misbehaviourMsg);
```

Comment:
What is the reason for this change?

Reply:
What would be the reason to emit misbehaviorMsg?

Reply:
Probably not needed, just wondering if there was a reason for this change now?

Reply:
Ah, I had updated some events in the PR but I ended up removing them.